packages/testing-helpers/src/scopedElementsWrapper.js

```diff
@@ -0,0 +1,64 @@
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { html, LitElement, TemplateResult } from 'lit-element';
+import { nothing } from 'lit-html';
+import { isIterable } from './lib.js';
+
+const transform = template => {
+  if (isIterable(template)) {
+    return [...template].map(v => transform(v));
+  }
+
+  if (template instanceof TemplateResult) {
+    return html(template.strings, ...template.values);
+  }
+
+  return template;
+};
+
+class ScopedElementsClass extends ScopedElementsMixin(LitElement) {
```

Comment:
small nit: since this is actually an element, maybe a name like `ScopedElementsHost`?

Reply:
I just changed to `ScopedElementsTestWrapper` which I think is more aligned with what is doing.